submissions/UA-girl/task_js_oop_exersise/index.js

```diff
@@ -0,0 +1,60 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/UA-girl/a-tiny-JS-world
+   Web app: https://ua-girl.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Creature {
+    constructor(species, name, age, gender, say) {
+        this.species = species;
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.say = say;
+    }
+
+    getDescription() {
```

Comment:
Check the magic [`toString` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString)

It is defined for every object in JS, it is overridable and called implictly in string context.
`print` ensures string context for its first argument.

Reply:
Fixed. Waiting for more advices )))